package.json

```diff
@@ -102,7 +103,8 @@
   },
   "jest": {
     "roots": [
-      "<rootDir>/src"
+      "<rootDir>/src",
+      "<rootDir>/server"
     ],
     "collectCoverageFrom": [
```

Comment:
Could you please also include `"server/**/*.js"` here? It's needed to collect coverage from all files inside of the server folder